app/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Contact;
+use Auth;
+
+class ContactController extends Controller
+{
+
+    function store(Request $request)
+    {
+        // dd($request);
+        $contact = new Contact();
+
+        $contact -> name = $request -> name;
+        $contact -> mail = $request -> mail;
+        $contact -> content = $request -> message;
+
+        $contact -> save();
+
+        return redirect()->route('contact.create');
```

Comment:
`contact.create` というルーティング名のものはないので、純粋にviewファイルをreturnしてあげてください
```suggestion
        return view('contact.create');
```